script.js

```diff
@@ -1,4 +1,218 @@
+
+// function to generate random number
+function random(max) {
+  let randomNum = Math.floor(Math.random() * max) + 1;
+  return randomNum;
+}
+// create function to generate random item
+function randomOption() {
+  let randomItemNum = random(3);
+  let item;
+  if (randomItemNum == 1) {
+    item = 'scissors';
+  }
+  else if (randomItemNum == 2) {
+    item = 'paper';
+  }
+  else if (randomItemNum == 3) {
+    item = 'stone';
+  }
+  return item;
+}
+
+//**STAGE-2 Win-Loss Record */
```

Comment:
Try to stick to either inline (//) or block (/* */) comment format. No need to have both.